example/bin/conf_test.cpp

```diff
@@ -0,0 +1,26 @@
+#include <hocon/parser/config_document_factory.hpp>
```

Comment:
This should be moved to the `lib/test` directory and integrated with our test suite.